GNUmakefile

```diff
@@ -244,3 +250,466 @@ deploy-images:
 	@IMAGE=$(ENGINE_IMG) ./build/push
 	@IMAGE=$(PAIR_IMG) ./build/push
 	@IMAGE=$(DS_IMG) ./build/push
+	@echo ""
+	@echo "Pushing developer image...."
+	@docker push $(DEV_IMAGE)
+
+
+KUBECONFIG?=
+INTERNAL_CONFIG:=$(KUBECONFIG:$(HOME)%=%)
+
+SRC_CONTEXT?=kind-src-cluster
+DST_CONTEXT?=kind-dst-cluster
+
+SRC_ES_NODE_PORT?=
+DST_ES_NODE_PORT?=
+
+SRC_CONTROL_PANE:=$(shell /bin/bash -c "kubectl get pod -n kube-system  -o name --context=$(SRC_CONTEXT)| grep kube-apiserver")
+SRC_CLUSTER_IP=$(shell (kubectl get -n kube-system $(SRC_CONTROL_PANE) -o yaml --context=$(SRC_CONTEXT)| grep advertise-address= | cut -c27-))
+DST_CONTROL_PANE:=$(shell /bin/bash -c "kubectl get pod -n kube-system  -o name --context=$(DST_CONTEXT)| grep kube-apiserver")
+DST_CLUSTER_IP=$(shell (kubectl get -n kube-system $(DST_CONTROL_PANE) -o yaml --context=$(DST_CONTEXT)| grep advertise-address= | cut -c27-))
+
+export MINIO_ACCESS_KEY=not@id
+export MINIO_SECRET_KEY=not@secret
+
+# Register the Kubemove crds in both source and destination cluster
+# Example: make register-crds SRC_CONTEXT=<source context> DST_CONTEXT=<destination context>
+.PHONY: register-crds
+register-crds:
+	@echo "Registering the CRDs in both source and destination clusters...."
+	@docker run                                                          \
+			-i                                                           \
+			--rm                                                         \
+			--net=host                                                   \
+			-u $$(id -u):$$(id -g)                                       \
+			-v $$(pwd):/go/src/$(MODULE)                                 \
+			-v $$(pwd)/.go/cache:/.cache                                 \
+			-v $(HOME)/.kube:/.kube                                      \
+			-w /go/src/$(MODULE)                                         \
+			--env HTTP_PROXY=$(HTTP_PROXY)                               \
+			--env HTTPS_PROXY=$(HTTPS_PROXY)                             \
+			--env GO111MODULE=on                                         \
+			--env GOFLAGS="-mod=vendor"                                  \
+			--env KUBECONFIG=$(INTERNAL_CONFIG)                          \
+			$(DEV_IMAGE)                                                 \
+			/bin/bash -c '                                               \
```

Comment:
This will not change anything locally, right? We can run this command without docker also.

Reply:
For other kubemove installation command also.

Reply:
removed